zkmemory/examples/256bits-machine.rs

```diff
@@ -223,7 +226,7 @@ where
             memory: RBTree::new(),
             memory_allocated: config.memory,
             word_size: config.word_size,
-            time_log: 0,
+            time_log: 1,
```

Comment:
I also changed the starting value of `time_log` from $0$ to $1$, since the `time_log` value from the proposal and the project description starts from $1$, while `stack_depth` starts from $0$.